consensus/parlia/parlia.go

```diff
@@ -1275,6 +1278,7 @@ func (p *Parlia) systemCall(from, contract libcommon.Address, data []byte, ibs *
 		msg.Gas(),
 		msg.Value(),
 		false,
+		firehoseContext,
```

Comment:
Any reason why this is added?

Reply:
Aah, that was added coz I also made dependent changes in other pkg, but then realised i was not supposed to make that, so forgot to remove this. 

Removed.